src/components/HdResponsive.vue

```diff
@@ -18,42 +16,46 @@ export default {
     return {
       matches: {},
       indeterminate: !matchMediaAvailable,
+      listeners: {},
     };
   },
   watch: {
     breakpoints() {
-      this.setResponsiveFlags();
+      this.setListeners();
     },
   },
-  created() {
-    this.setResponsiveFlags();
-  },
-  mounted() {
-    this.addResizeEvents();
-  },
-  beforeDestroy() {
-    this.removeResizeEvents();
+  beforeMount() {
+    this.setListeners();
   },
   methods: {
-    setResponsiveFlags() {
+    /* istanbul ignore next */
```

Comment:
I don't think you need to ignore it in Istanbul in this case

Reply:
I need it to have _updateMatch_ not counted on the coverage. Unfortunately, I can no test it because of the differences between the CLI and the browser environment. Same for the listeners.